build_tools/aws-sdk-code-generator/spec/interfaces/plugins/endpoints_spec.rb

```diff
@@ -124,6 +125,10 @@
         EndpointsPrecedence::Client.new(stub_responses: true, region: 'config')
       end
 
+      before(:each) do
```

Comment:
Is this still necessary? If so how can we avoid doing this?

Reply:
Added a comment.  This is neccessary because config after_initialize will call the endpoint resolver once.  This lets us achieve that before the actual tests with their assertions about how the resolver/parameters are called.